.github/workflows/rustfmt.yml

```diff
@@ -0,0 +1,16 @@
+name: Rustfmt
+
+on:
+  push:
+   branches: [master]
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install nightly toolchain
+        run: rustup toolchain install nightly
+      - name: Install nightly rustfmt
+        run: rustup component add rustfmt --toolchain nightly
+      - name: Run rustfmt
+        run: cargo +nightly fmt --all -- --check
```

Comment:
Wouldn't it perhaps make more sense to add this to the existing `ci.yml`? :)

Or, or as long as there are also direct pushes to master then perhaps it makes sense to have both. A check on pull requests and this for direct pushes.

Reply:
Yeah I wasn't sure where and when this should run. I can merge it into the existing `ci.yml`!